packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -129,18 +142,33 @@ class EmbeddedContentConsumer extends React.Component {
       onAuthorize,
       options,
       eventHandlers,
+      intl,
       ...customProps
     } = this.props;
 
     return (
-      <div
-        {...customProps}
-        ref={(element) => { this.embeddedContentWrapper = element; }}
-      />
+      <div className={cx('embedded-content-container')}>
```

Comment:
Is this new container necessary?  Can't the hidden text be siblings of the existing container?

Reply:
I was initially returning this in a fragment but I ran into the issue I mentioned above. I added this new container instead so that I could apply those style changes to resolve that issue.